pandas/tests/io/test_excel.py

```diff
@@ -56,7 +56,24 @@ def ignore_xlrd_time_clock_warning():
         yield
 
 
-@pytest.mark.parametrize("ext", ['.xls', '.xlsx', '.xlsm'])
+@pytest.fixture
+def df_ref():
+    """
+    Obtain the reference data from read_csv with the Python engine.
+    """
+    df_ref = read_csv('test1.csv', index_col=0,
+                      parse_dates=True, engine='python')
+    return df_ref
+
+
+@pytest.fixture(params=['.xls', '.xlsx', '.xlsm'])
+def read_ext(request):
```

Comment:
The majority of this PR is a result of renaming this fixture so as not to conflict with the valid extensions for writing (which don't include macro files AFAIK)